src/pages/index.tsx

```diff
@@ -74,23 +34,26 @@ type FilledBubble = Bubble & {
   };
 };
 
-type Props = {
-  bubbles: FilledBubble[];
-  labels: Label[];
-};
-
-const HomePage: React.FC<Props> = (props: Props) => {
+const HomePage: React.FC = () => {
+  const [ labels, setLabels ] = useState<Label[]>([]);
   const [ bubbles, setBubbles ] = useState<FilledBubble[]>([]);
-  const [ isBubbleDetailsVisible, setIsBubbleDetailsVisible ] = useState(false);
   const [ oppenedBubbleId, setOppenedBubbleId ] = useState(null);
+  const [ isBubbleDetailsVisible, setIsBubbleDetailsVisible ] = useState(false);
   const [ isNewBubbleModalVisible, setIsNewBubbleModalVisible ] = useState(false);
 
+  const { data: bubblesData } = useFetch('/bubbles');
+  const { data: labelsData } = useFetch('/labels');
+
   useEffect(() => {
-    setBubbles(props.bubbles.map(bubble => ({
-      ...bubble,
-      createdAt: new Date(bubble.createdAt),
-    })));
-  }, []);
+    if(bubblesData && labelsData !== undefined) {
+      setBubbles(bubblesData.map(bubble => ({
+        ...bubble,
+        createdAt: new Date(bubble.createdAt),
+      })));
+
```

Comment:
o carregamento acontece de forma independente.
```suggestion
if(labelsData !== undefined)
```